core/src/main/java/org/openmbee/sdvc/core/builders/PermissionUpdateResponseBuilder.java

```diff
@@ -0,0 +1,155 @@
+package org.openmbee.sdvc.core.builders;
+
+import org.openmbee.sdvc.core.objects.PermissionUpdateResponse;
+import org.openmbee.sdvc.data.domains.global.*;
+
+import java.util.*;
+
+public class PermissionUpdateResponseBuilder {
+
+    private class PermissionUpdateWrapper {
```

Comment:
I don't think the wrapper here is needed? Since PermissionUpdate only has strings and enums the java default equals and hashcode should be enough for the hashsets